guides/contribution.md

```diff
@@ -0,0 +1,119 @@
+# Contribution guide
+
+Welcome to the Mística Design System contribution guide for external teams.
+
+As an external team, you have the opportunity to contribute to the Mística Design System by sharing your ideas and expertise.
+Whether you are a designer, developer, or product manager, your contributions can help us improve the system and make it more useful for a wider range of users.
+
+This guide is designed to help you understand how you can contribute to Mística, what kind of contributions we are looking for, and how to make sure your contributions are consistent with our way of working.
```

Comment:
:robot: OpenAI

The paragraph provides a good overview of the purpose of the guide. However, it could be more specific about the benefits of contributing to the Mística Design System.
```suggestion
As an external team, you have the opportunity to contribute to the Mística Design System by sharing your ideas and expertise. Your contributions can help us improve the system and make it more useful for a wider range of users. This guide is designed to help you understand how you can contribute to Mística, what kind of contributions we are looking for, and how to make sure your contributions are consistent with our way of working.
```

<!-- This is an auto-generated comment by OpenAI -->